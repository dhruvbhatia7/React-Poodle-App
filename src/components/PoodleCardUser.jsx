import React from 'react';
import './PoodleCard.css';


class PoodleCardUser extends React.PureComponent {
    render() {
        return (
           <div className="poodle-details">
                <img className="poodle-img" alt="" src={this.props.image}/>
                <div className="poodle-name">{this.props.name}</div>  
            </div>

        ); 
    }
}

export default PoodleCardUser;