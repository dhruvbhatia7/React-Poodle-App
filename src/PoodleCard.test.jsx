import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallowToJson } from 'enzyme-to-json';

import PoodleCardUser from './components/PoodleCardUser';

configure({adapter: new Adapter() });


describe('PoodleCard Name', () => {
    it ('should render correctly', () => {
        const output = shallow(<PoodleCardUser name="Swaffy" />);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});

describe('PoodleCard img', () => {
    it ('img src should be correct', () => {
        const output = shallow(<PoodleCardUser image="https://bull" />);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});