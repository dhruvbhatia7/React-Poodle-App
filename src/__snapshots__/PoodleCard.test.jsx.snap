// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PoodleCard Name should render correctly 1`] = `
<div
  className="poodle-details"
>
  <img
    alt=""
    className="poodle-img"
  />
  <div
    className="poodle-name"
  >
    Swaffy
  </div>
</div>
`;

exports[`PoodleCard img img src should be correct 1`] = `
<div
  className="poodle-details"
>
  <img
    alt=""
    className="poodle-img"
    src="https://bull"
  />
  <div
    className="poodle-name"
  />
</div>
`;
